Snippets/Snippets_6/DataBus/CleanupStrategy/Define.cs

```diff
@@ -0,0 +1,16 @@
+﻿namespace Snippets6.DataBus.CleanupStrategy
+{
+    using NServiceBus;
+
+    public class Define
+    {
+        public Define()
+        {
+            #region DefineFileLocationForDatabusFiles
+            EndpointConfiguration endpointConfiguration = new EndpointConfiguration("endpointName");
+            endpointConfiguration.UseDataBus<FileShareDataBus>().BasePath(@"c:\database_files\");
```

Comment:
Definitely databus instead of database! Good catch! And I see the current doco mentioned attachments, I'll rename that as well.